src/minitest.cr

```diff
@@ -67,28 +98,38 @@ module Minitest
 
   def self.run(args = nil)
     process_args(args) if args
-    reporter.start
+    puts options
```

Comment:
Debug leftover?

Reply:
No. Without this print, we wouldn't know the run options, especially the default seed; the filters are important, too; to know how to reproduce a random failure: just copy paste the options.